buildpack/bomfile.go

```diff
@@ -0,0 +1,105 @@
+package buildpack
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/buildpacks/lifecycle/buildpack/layertypes"
+)
+
+const (
+	LayerTypeBuild LayerType = iota
+	LayerTypeCache
+	LayerTypeLaunch
+)
+
+type LayerType int
+
+type BOMFile struct {
+	BuildpackID string
+	LayerName   string
+	LayerType   LayerType
+	Path        string
+}
+
+func (b *BOMFile) Name() (string, bool) {
```

Comment:
```suggestion
func (b *BOMFile) BOMType() (string, bool) {
```
This seems more accurate... could we clarify what the second return value is supposed to mean?

Reply:
Or maybe `BOMFormat`?

Reply:
Actually maybe `Name` is fine. Perhaps we could add a docs comment to make it clearer.

Reply:
_Fixed in latest commit._